### PR TITLE
MOYO header

### PR DESCRIPTION
**Pull request by blacktox96**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,22 +1,100 @@
...
+          </li>
+          <li class="nav__li">
+            <a  href="photo.html"
+                class="nav__url"
+            >PHOTO</a>
+          </li>
+          <li class="nav__li">
+            <a  href="video.html"
+                class="nav__url"
+            >VIDEO</a>
```
Comment (etojeDenys) [inline src/index.html:89]: image

File: src/index.html [hunk 2]
```diff
@@ -1,22 +1,100 @@
...
+      >
+        <img
+          class="header__logo"
+          src="moyo_1.png"
+          alt="header_logo"
+        >
+      </a>
+      <!-- nav.nav>ul.nav__list>li*8.nav__li>a.nav__url -->
+      <nav class="nav">
+        <div class="active-underline"></div>
```
Comment (etojeDenys) [inline src/index.html:48]: try to implement this task without using the tag for underline 
```suggestion
```

File: src/index.html [hunk 3]
```diff
@@ -1,22 +1,100 @@
...
+            >VIDEO</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <h1 class="h1">Moyo header</h1>
+    <main>
+    </main>
+    <footer>
+    </footer>
```
Comment (etojeDenys) [inline src/index.html:98]: we don't need that

File: src/style.css [hunk 4]
```diff
@@ -1,3 +1,185 @@
...
+
+.is-active::after {
+  content: '';
+  display: block;
+  background-color: #00ACDC;
+  height: 4px;
+  /* ------------------------------------------- */
+  width: 37px;
+  /* Impossible to set same length in pixels as url-word length */
+  /* ------------------------------------------- */
```
Comment (etojeDenys) [inline src/style.css:177]: remove all comments

File: src/style.css [hunk 5]
```diff
@@ -1,3 +1,185 @@
...
+  font-style: normal;
+}
+
+.roboto-thin-italic {
+  font-family: "Roboto", sans-serif;
+  font-weight: 100;
+  font-style: italic;
+}
+
+.roboto-light-italic {
```
Comment (etojeDenys) [inline src/style.css:89]: do you use those classes? looks redundant

**Review by etojeDenys — CHANGES_REQUESTED** [review]
1. you don't need to import the image, check the img or images folder
image


2. provide the links from readme.md to your PR description
3. feel free to ask for some help in the chat

File: src/style.css (hunk 5 above)
Comment (blacktox96) [reply src/style.css:89]: Just copied that Google offered to myself

> вы используете эти классы? выглядит избыточным

File: src/index.html (hunk 2 above)
Comment (blacktox96) [reply src/index.html:48]: > try to implement this task without using the tag for underline

Made

**Comment by blacktox96** [comment]
> 1. you don't need to import the image, check the img or images folder
> 
> image
> 2. provide the links from readme.md to your PR description
> 3. feel free to ask for some help in the chat

1. made
2. didnt understood what is "PR" and what you mean
3. 🙏

File: src/style.css (hunk 4 above)
Comment (blacktox96) [reply src/style.css:177]: Ok

Comment (blacktox96) [reply src/style.css:177]: But what a solution to make a length same to url-word length without JavaScript?

File: src/index.html (hunk 1 above)
Comment (blacktox96) [reply src/index.html:89]: Seems right

File: src/index.html (hunk 3 above)
Comment (blacktox96) [reply src/index.html:99]: Removed.

**Comment by blacktox96** [comment]
@etojeDenys HELP 😢 
i ahve polished all complains of linter, fixed all that you said, but that test still doesnt like my code 😩

**Pull request closed without merging**